Add `How Zetera became a Medal Hunter` news post

0x84f opened the pull request:
I am not sure how to name this, I'm open to ideas! Once the name is figured out I will ask flyte for the banner in the Discord.
The imgur link will need to be replaced by an assets.ppy.sh link.

## Self-check
- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] ~~*(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)~~

Tanza3D commented:
gave this a read; little thing i noticed is that at the bottom, you say that ephemeral teased new hush-hush and beatmap pack medals. in reality, only the hush-hush was teased, and our[osu! medal hunters] server members, more specifically chromb, were able to guess the beatmap pack, and later on seasonal pack medal image urls. not that big of an issue but just wanted to say something about it.

beyond that, it all seems to read well, and definitely seems good! thought i'd leave my thoughts on it here.

0x84f commented:
> gave this a read; little thing i noticed is that at the bottom, you say that ephemeral teased new hush-hush and beatmap pack medals. in reality, only the hush-hush was teased, and our[osu! medal hunters] server members, more specifically chromb, were able to guess the beatmap pack image urls, and later on someone else was able to guess the seasonal pack medal image urls. not that big of an issue but just wanted to say something about it.
> 
> beyond that, it all seems to read well, and definitely seems good! thought i'd leave my thoughts on it here.

Thanks, I changed it to the "Osekai community" since that feels better than to just say chromb and you said "someone else" found seasonal pack medals

Tanza3D commented:
> > gave this a read; little thing i noticed is that at the bottom, you say that ephemeral teased new hush-hush and beatmap pack medals. in reality, only the hush-hush was teased, and our[osu! medal hunters] server members, more specifically chromb, were able to guess the beatmap pack image urls, and later on someone else was able to guess the seasonal pack medal image urls. not that big of an issue but just wanted to say something about it.
> > beyond that, it all seems to read well, and definitely seems good! thought i'd leave my thoughts on it here.
> 
> Thanks, I changed it to the "Osekai community" since that feels better than to just say chromb and you said "someone else" found seasonal pack medals

yep, seems better now! *may* be worth changing it to the osu! Medal Hunters community, as Osekai is just a website which community weirdly resides within that community, but works either way! thanks for the fix!

0x84f commented:
> > > gave this a read; little thing i noticed is that at the bottom, you say that ephemeral teased new hush-hush and beatmap pack medals. in reality, only the hush-hush was teased, and our[osu! medal hunters] server members, more specifically chromb, were able to guess the beatmap pack image urls, and later on someone else was able to guess the seasonal pack medal image urls. not that big of an issue but just wanted to say something about it.
> > > beyond that, it all seems to read well, and definitely seems good! thought i'd leave my thoughts on it here.
> > 
> > 
> > Thanks, I changed it to the "Osekai community" since that feels better than to just say chromb and you said "someone else" found seasonal pack medals
> 
> yep, seems better now! _may_ be worth changing it to the osu! Medal Hunters community, as Osekai is just a website which community weirdly resides within that community, but works either way! thanks for the fix!

Makes sense yeah, changed that to Medal Hunters community and also did some reference shifting to better fit that name within the article

Walavouchey commented on news/2022-09-01-how-zetera-became-a-medal-hunter.md:
```diff
@@ -0,0 +1,55 @@
+
+You might ask yourself what Zetera meant when he said that he kept "dropping out of the 90% medals club". Well, in 2019 the medal hunter community created [Osekai](https://osekai.net/home/), a website made for alternative rankings, in-depth profile information, medal solutions etc. On their [Discord server](https://discord.com/invite/8qpNTs6) there are "clubs" that people "join" by receiving a Di ...
+
+If you are interested in medal hunting definitely look into Osekai and join the Medal Hunters Discord server, and definitely stay on the lookout for new medals as Ephemeral teased new [hush-hush](https://twitter.com/osekaiapp/status/1564610856381464580) the community found [some more medals](https://twitter.com/osekaiapp/status/1564871086617071616) by guessing the links to them!
```
dunno if the last part should be in the article if they haven't been released or teased yet (@Ephemeralis)

```suggestion
If you are interested in medal hunting definitely look into Osekai and join the [Medal Hunters Discord server](https://discord.com/invite/8qpNTs6), and definitely stay on the lookout for new medals, especially since Ephemeral has just teased [a new Hush-Hush medal](https://twitter.com/osekaiapp/status/1564610856381464580) and with the community discovering [even more upcoming medals](https://twitter.com/osekaiapp/status/1564871086617071616) by finding their assets on the website.
```

0x84f replied:
i mean it was tweeted publicly so I guess it'd be a open secret?

Walavouchey replied:
doesn't mean it should be on a news post. just mentioning this because there was a [pr](https://github.com/ppy/osu-wiki/pull/5736) that was going to add hush-hush medals and their known unlock requirements to the wiki, but they had to be removed even though there is a whole community that knows this stuff, which in itself is fine

Ephemeralis replied:
i'd avoid mentioning the fact hint solutions are available there outright, though phrasing it as a community of people interested in medals should be fine

0x84f replied:
Wala's concerns were addressed by [Eph ](https://discord.com/channels/188630481301012481/218677502141399041/1016089780842406031)in the Discord, as for what Eph said it has been applied in [bc9d0cc](https://github.com/ppy/osu-wiki/pull/7949/commits/bc9d0cc8445cc93145a4ab04ce28e99991fe7269)

Walavouchey commented on news/2022-09-01-how-zetera-became-a-medal-hunter.md:
```diff
@@ -0,0 +1,55 @@
+
+If you are interested in medal hunting definitely look into Osekai and join the Medal Hunters Discord server, and definitely stay on the lookout for new medals as Ephemeral teased new [hush-hush](https://twitter.com/osekaiapp/status/1564610856381464580) the community found [some more medals](https://twitter.com/osekaiapp/status/1564871086617071616) by guessing the links to them!
+
+Lastly, the full conversation I had with Zetera goes into much more detail about hunting medals in general, and I highly suggest you [check it out](https://gist.github.com/0x84f/5b93172d0ca1c3bf975eb8b40faa3474) if this article was interesting to you.
```
```suggestion
Lastly, the full conversation with Zetera goes into much more detail about hunting medals in general, and I highly suggest you [check it out](https://gist.github.com/0x84f/5b93172d0ca1c3bf975eb8b40faa3474) if this article was interesting to you.
```

Walavouchey commented on news/2022-09-01-how-zetera-became-a-medal-hunter.md:
```diff
@@ -0,0 +1,55 @@
+
+Despite all of the above, the community managed to figure out the solutions to all of the new medals! Well, all but one.. "The Girl in the Forest" stood unclaimed. They deduced that the map that needed to be played was [Takuya](https://osu.ppy.sh/users/342372)'s set of [Pika Girl](https://osu.ppy.sh/beatmapsets/40440#osu/128718) by [S3RL](https://osu.ppy.sh/beatmaps/artists/9). Zetera immediately ...
+
+> It's been roughly a week since the Halloween medal release, and just about every medal outside of one has been conclusively claimed and figured out.
+> Originally, I thought that the Firmament Moves would be the baffler for most people. Apparently not!
+> So, let's have a little fun with this, and up the ante a little bit.
```
i'm pretty sure these paragraphs should be spaced like so but i also remember that remark complains about it <.< (@cl8n ?)
```
> ...
> 
> ...
>
> ...
```

0x84f replied:
Discussed on [Discord ](https://discord.com/channels/188630481301012481/218677502141399041/1016278123056218182)and applied via [816c1f7](https://github.com/ppy/osu-wiki/pull/7949/commits/816c1f78326548bd1506e6a0976cf9f42b46e469)

Walavouchey commented on news/2022-09-01-how-zetera-became-a-medal-hunter.md:
```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "How Zetera became a Medal Hunter"
+date: 2022-09-31 00:00:00 +0000
```
ah, yes, september 31

0x84f replied:
just a placeholder lol didnt even realize what I wrote, I will change it later

Walavouchey reviewed (changes requested):
there's no need to correct spelling or formatting when quoting (looking at https://github.com/ppy/osu-wiki/pull/7949/commits/1dffbb63768c7ce26c557833f4f3c68457ab0a81)

0x84f commented on news/2022-09-01-how-zetera-became-a-medal-hunter.md:
```diff
@@ -0,0 +1,54 @@
+
+Join us for a history lesson and a story about an elusive medal and one man's journey to unravel it's secret!
+
+Before we begin with this story, we first must learn about the history of medals. Medals were first introduced in early August 2008, with their original goal being to serve as an achievement system that could (but never did) become a part of osu!'s "story mode". The first ever medals to get added were the playcount medals and since they were retroactively given to all users that already had the n ...
```
they are just two examples

0x84f commented on news/2022-09-01-how-zetera-became-a-medal-hunter.md:
```diff
@@ -0,0 +1,54 @@
+
+So how did he get into medal hunting? It was quite simple actually, he just didn't like how unobtained medals looked like on the old websites (he described them as "huge boxes with a question mark") and would rather have had those boxes show the actual hand-drawn designs than just a question mark, older players might remember them looking like this:
+
+![](https://i.imgur.com/ca5Z33V.png)
```
I thought it could sit there as a temporary image until it's uploaded to assets.ppy.sh, but I will upload it if you don't think its a good idea to have it hotlinked at all

Walavouchey replied:
putting it in the repo is how it's usually done yes

0x84f replied:
Done in [224a88b](https://github.com/ppy/osu-wiki/pull/7949/commits/224a88b688ace426e53868ec455962c93c22b8c3)

0x84f commented on news/2022-09-01-how-zetera-became-a-medal-hunter.md:
```diff
@@ -0,0 +1,54 @@
+> Originally, I thought that the Firmament Moves would be the baffler for most people. Apparently not!
+> So, let's have a little fun with this, and up the ante a little bit.
+
+And "up the ante a little bit" he did. A bounty was set on the medal, stating that the first person who unlocks the medal and messages him with the correct set of criteria required to unlock the medal would receive a "Medal Hunter" user title, which would the first of it's kind, and to this day would remain as such.
```
Can you edit out the `the` in the 2nd sentence and I will add it to the batch

Walavouchey replied:
woops

0x84f commented on news/2022-09-01-how-zetera-became-a-medal-hunter.md:
```diff
@@ -0,0 +1,54 @@
+
+> Hush-Hush medals are "secret" medals awarded for pulling off very specific feats, which are not explicitly told to the player (hence the name). These feats can range anywhere from completing a beatmap under peculiar circumstances to setting a score with an unusual mod combination.
+
+Throughout the years new hush-hush medals were extremely rare, and it was only in late 2016 that medals started being released in batches after [this newspost](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals) and since then would start receiving semi-regular additions.
```
From my research they are just released sporadically, no special timing behind them (except maybe for Halloween?)

Walavouchey replied:
which is why i asked for any sort of average. maybe how many batches there have been so far could be mentioned instead (again i'm just suggesting)

0x84f replied:
https://osu.ppy.sh/home/news/2016-10-29-halloween-cometh
https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals
https://osu.ppy.sh/home/news/2018-10-23-halloween-2018-descends
https://osu.ppy.sh/home/news/2019-10-31-osu-spooktacular-halloween-special

2016: 2
2017: 0
2018: 1
2019: 1
All but the 1 additional batch in 2016 were added on Halloween, and @Ephemeralis teased that medals would be coming this month, breaking the streak.

Walavouchey commented:
> I am not sure how to name this, I'm open to ideas!

tbh i think the title is perfect to the point

0x84f commented on news/2022-09-01-how-zetera-became-a-medal-hunter.md:
```diff
@@ -0,0 +1,54 @@
+
+Since getting batches of medals was still new, there wasn't a lot of knowledge behind what made the medal system tick, especially because the whole system was rewritten in 2016. Nowadays the community knows exactly what to look for and how extensive the system is, but back then the community's ideas were much more abstract than what was actually possible, simply because it was really hard to gaug ...
+
+Despite all of the above, the community managed to figure out the solutions to all of the new medals! Well, all but one.. "The Girl in the Forest" stood unclaimed. Zetera and the community first deduced that the map that needed to be played was [Takuya](https://osu.ppy.sh/users/342372)'s set of [Pika Girl](https://osu.ppy.sh/beatmapsets/40440#osu/128718) by [S3RL](https://osu.ppy.sh/beatmaps/arti ...
```
I was flip-flopping between "the community" and "Zetera and the community", but I think "the community" is more appropriate

0x84f replied:
I applied your review in [07fa898](https://github.com/ppy/osu-wiki/pull/7949/commits/07fa898231484fa92e69b14b6ab836f42e6e33f5) along with changing it to the community and adding ```Zetera then immediately went and SS'd the map, but that didn't work.```

0x84f commented:
I asked flyte for the banner and [they delivered](https://discord.com/channels/188630481301012481/218677502141399041/1016621650315317249), just asked them to make some final changes and the news post is ready (Eph needs to review as well)

Ephemeralis commented:
I've tidied up a bit of prose and expanded on some terminology (like mentioning flair text and a few other things).

We can schedule this for sometime this week (thursday/friday/sunday) if people think it's in a good spot. Still needs a banner added to it though!

Walavouchey commented on news/2022-09-01-how-zetera-became-a-medal-hunter.md:
```diff
@@ -0,0 +1,79 @@
+
+Despite these challenges, the community managed to figure out the solutions to all of 2017's new medals save for one: "The Girl in the Forest". The community first deduced that the relevant map was [Takuya](https://osu.ppy.sh/users/342372)'s set of [Pika Girl](https://osu.ppy.sh/beatmapsets/40440#osu/128718) by [S3RL](https://osu.ppy.sh/beatmaps/artists/9) after linking the hints and flair text b ...
+
+The limits of the medal system needed to be examined more closely, albeit with educated guesswork and deduction. Surely it can't check when you lost your full combo or when an SS was lost, so the only things left would be the miss count, max combo, counts of 300s, 100s, 50s and misses, playing in different modes and a couple of other things, such as at what time was the score submitted. At the ti ...
```
```suggestion
The limits of the medal system needed to be examined more closely, albeit with educated guesswork and deduction. Surely it can't check when you lost your full combo or when an SS was lost, so the only things left would be the miss count, max combo, counts of 300s, 100s, 50s and misses, playing in different modes and a couple of other things, such as at what time was the score submitted. At the time, the flair text read as "First Generation in more than one ways", which lead Zetera to think that it might be related to Pokémon or perhaps the first generation of influential mappers, but in the end, he found himself stumped.
```

Walavouchey commented on news/2022-09-01-how-zetera-became-a-medal-hunter.md:
```diff
@@ -0,0 +1,79 @@
+
+![](https://assets.ppy.sh/medals/web/all-secret-lightless.png) ![](https://assets.ppy.sh/medals/web/all-secret-deliberation.png) ![](https://assets.ppy.sh/medals/web/all-secret-when-you-see-it.png)
+
+If puzzling over flairs and hints isn't your thing, there are **3** beatmap packs to achieve as well just by finishing the packs linked to them. Check them out below (and you can even click on an image to be taken to its respective pack!):
```
```suggestion
If puzzling over flairs and hints isn't your thing, there are **3** beatmap packs to achieve as well just by finishing the maps in them. Check them out below (and you can even click on an image to be taken to the corresponding pack!):
```

Walavouchey commented on news/2022-09-01-how-zetera-became-a-medal-hunter.md:
```diff
@@ -0,0 +1,79 @@
+
+![https://osu.ppy.sh/beatmaps/packs/2483](https://assets.ppy.sh/medals/web/all-packs-aitsuki.png) ![https://osu.ppy.sh/beatmaps/packs/2482](https://assets.ppy.sh/medals/web/all-packs-maduk.png) ![https://osu.ppy.sh/beatmaps/packs/2481](https://assets.ppy.sh/medals/web/all-packs-vocaloid.png)
+
+Don't want to hunt for the medals alone? Join the Medal Hunters Discord server and check out Osekai *(please note that the website contains solutions to medals, so if you'd rather solve them yourselves the site might not be for you)*!
```
```suggestion
Don't want to hunt for the medals alone? Join the Medal Hunters Discord server and check out Osekai! *(Please note that the website contains solutions to medals, so if you'd rather solve them yourselves the site might not be for you.)*
```

Walavouchey commented on news/2022-09-01-how-zetera-became-a-medal-hunter.md:
```diff
@@ -0,0 +1,79 @@
+
+Since getting batches of medals was still new, there wasn't a lot of knowledge behind what made the medal system tick, especially since the whole thing was rewritten back in 2016. Nowadays, the community knows exactly what to look for and how extensive the system is, but back then the community's ideas were much more abstract than what was actually possible, simply because it was really hard to g ...
+
+Figuring out how to solve a medal is often a complicated process, as the only real hints provided are the medal's name, icon and some mouse-over text known internally as a 'flair'. At times, even these are not enough, and some medals have even had all three of these things changed over the years.
```
```suggestion
Figuring out how to solve a medal is often a complicated process, as the only real hints provided are the medal's name, icon and some mouse-over text known internally as a *flair*. At times, even these are not enough, and some medals have even had all three of these things changed over the years.
```

Walavouchey commented on news/2022-09-01-how-zetera-became-a-medal-hunter.md:
```diff
@@ -0,0 +1,79 @@
+
+After a gruelling slog of trial and error, he finally found himself staring at the fruits of his labor, and after sending a forum PM to Ephemeral with the correct answer, he was officially dubbed as osu!'s first Medal Hunter. He was really happy about receiving the medal, but he felt like there were other people in the community that deserved this medal more than him, and had to say this during o ...
+
+> User titles and I are a complicated topic. It's not really part of this interview's topic, but I did a lot of different things for the community in the hopes of leaving a mark somewhere(...), because if there was something I wanted more than anything else, it was the Elite Mapper title.
```
```suggestion
> User titles and I are a complicated topic. It's not really part of this interview's topic, but I did a lot of different things for the community in the hopes of leaving a mark somewhere [...] because if there was something I wanted more than anything else, it was the Elite Mapper title.
```

Walavouchey reviewed (dismissed):
remember to redate
